Source/WebKit/UIProcess/Automation/BidiScriptAgent.cpp

```diff
@@ -51,6 +55,103 @@ BidiScriptAgent::BidiScriptAgent(WebAutomationSession& session, BackendDispatche
 
 BidiScriptAgent::~BidiScriptAgent() = default;
 
+static RefPtr<Inspector::Protocol::BidiScript::RemoteValue> convertJSONToRemoteValue(const JSON::Value* jsonValue)
```

Comment:
The spec for [serialize as a remote value](https://w3c.github.io/webdriver-bidi/#serialize-as-a-remote-value) has a number of extra parameters alongside `value`, like `serializationOptions`, `realm`, etc. that are used to control and limit the serialization process. We could add them to this function, even if just as a placeholder for now.